src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,310 @@
+- [Ethereum Foundation](https://ethereum.foundation/)
+
+During this long developmental cycle, libp2p was running in the above validators on the Beacon Chain.
+And once the Beacon Chain was merged with the original execution, libp2p was fully integrated into the Ethereum mainnet!
+
+# How libp2p is used by Ethereum validators 🔍
+
+So which exact libp2p modules are utilized by validators?
+Per the specification, Ethereum validators use the following libp2p features:
+- [Transport](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#transport): TCP
+- [Encryption](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#encryption-and-identification): [Noise](https://github.com/libp2p/specs/tree/master/noise)
+- [Protocol Negotiation](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#protocol-negotiation): [Multistream Select 1.0](https://github.com/multiformats/multistream-select/#protocol)
+- [Stream Multiplexing](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#multiplexing): [mplex](https://github.com/libp2p/specs/tree/master/mplex) or [yamux](https://github.com/hashicorp/yamux/blob/master/spec.md)
+- [Messaging](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#the-gossip-domain-gossipsub): [GossipSub v1.1](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md)
+
+All of the modules above aside from yamux are mandatory.
+One notable omission is the [libp2p Kademlia DHT](https://github.com/libp2p/specs/tree/master/kad-dht); The networking specification instead specifies the use of discv5.
+The rationale is [provided here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#why-are-we-using-discv5-and-not-libp2p-kademlia-dht).
+
+The specific design details are out of scope for this post but are [described in great detail here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#design-decision-rationale).
+Please check it out to see why TCP was chosen (and not QUIC), why Noise, why GossipSub etc.
+
+## Network Statistics 📊
+
+Currently there are over [495,000 validators](https://beaconcha.in/validators) with this [current client diversity](https://clientdiversity.org/#distribution).
+libp2p is running on all those consensus clients; an amazing testament to 6+ years of hard work!
+
+# What's next? :rocket: 
+
+Now that the Merge is gone smoothly, the main collaboration efforts have resolved successfully.
+At this time, Protocol Labs is supporting the Ethereum community in ensuring the libp2p is running smoothly and reliably.
+Some of the future roadmap items for libp2p and Ethereum have been hinted in the specification (like exploring the usage of QUIC or an updated protocol for negotiation.)
```

Comment:
Can we not mention Protocol Select here? We've pushed out our plans for this improvement to the indefinite future, since the benefits are minimal at best.